server/serverAPI/model/food/requestSchema/InventoryIngredientSchema.ts

```diff
@@ -1,6 +1,7 @@
-import { IsNotEmpty, IsNumber, IsPositive, IsString, validate, ValidateNested } from "class-validator";
+import { IsArray, IsNotEmpty, IsNumber, IsObject, IsPositive, IsString, validate, ValidateNested } from "class-validator";
```

Comment:
It doesn't look like `IsObject` is used in this file

Reply:
it doesn't :relieved: